src/Flysystem/PublicAssetAdapter.php

```diff
@@ -53,6 +53,7 @@ protected function findRoot($root)
      */
     public function getPublicUrl($path)
     {
+        $path = Convert::slashes($path, '/');
```

Comment:
If this *is* the right place to do this, and a filesystem path is what we expect to be passed in here, please use `Path::normalise()` instead.

Reply:
As noted above, this method is for creating URLs, not filesystem paths

Path::nortmalise() will call Convert::slashes() will convert forward and backslashes to use the php constant `DIRECTORY_SEPARATOR` which will be set to the correct slash of the operating system

Reply:
Ahh, I didn't look as closely as I should have to that implementation. Good spot.